### PR TITLE
Bugfix #110

### PR DESCRIPTION
**Pull request by rpitingolo**

### CODE REVIEW

**Comment by rpitingolo** [comment]
@lhendey can you please review to make sure the output looks like what we expect?

**Comment by lhendey** [comment]
@rpitingolo - i can but not until at least 3:30.

**Comment by rpitingolo** [comment]
@lhendey ok thanks

File: Macros/ACS_summary_geo.sas [hunk 1]
```diff
@@ -213,6 +140,13 @@
 
 	   FamMarriedCouple: FamMaleholder: FamFemaleholder: Nonfamlivingalone: Nonfamnotlivingalone:
 
+	   FamIncomeLT75k_: FamIncomeGT200k_:
+	   
+       NumRenterCostBurden: NumRentSevereCostBurden: NumOwnerCostBurden: NumOwnSevereCostBurden:
+       RentCostBurdenDenom: OwnerCostBurdenDenom:
+
+       NumRentCst: NumOwnCstBurden:
```
Comment (lhendey) [inline Macros/ACS_summary_geo.sas]: Line 148 - repeated from line 130?

File: Macros/ACS_summary_geo.sas [hunk 2]
```diff
@@ -520,12 +483,31 @@
        mNumRentCstBurden_20_34K_&_years. mNumRentCstBurden_35_49K_&_years. 
        mNumRentCstBurden_50_74K_&_years. mNumRentCstBurden_75_99K_&_years. 
        mNumRentCstBurden_GT100K_&_years. 
-
-       mNumOwnCstBurden_LT10K_&_years. mNumOwnCstBurden_10_19K_&_years.
```
Comment (lhendey) [inline Macros/ACS_summary_geo.sas]: just double checking this section is in the new code?

File: Macros/ACS_summary_geo.sas [hunk 3]
```diff
@@ -573,63 +555,242 @@
 	   mhshldinc150000to199999_&_years.
 	   mhshldinc200000andover_&_years.
 
-           ;
+	   mAggFamilyIncome_&_years. 
+
+	   mFamIncomeLT75k_&_years. mFamIncomeGT200k_&_years.
+
```
Comment (lhendey) [inline Macros/ACS_summary_geo.sas]: i would prefer to see these vars put into groupings by topic.  - they'll be impossible to find later.

File: Macros/ACS_summary_geo_source_tr_vars.sas [hunk 4]
```diff
@@ -486,8 +504,8 @@
     mPop18_34YearsF_&_years.="Persons 18-34 years old, Female, MOE,&_years_dash "
     mPop35_64YearsM_&_years.="Persons 35-64 years old, Male, MOE,&_years_dash"
     mPop35_64YearsF_&_years.="Persons 35-64 years old, Female, MOE,&_years_dash"
-    mPop65andOverYearsM&_years.="Persons 65 years old and over, Male, MOE,&_years_dash "
-    mPop65andOverYearsF&_years.="Persons 65 years old and over, Female, MOE,&_years_dash "
+    mPop65andOverYearsM_&_years.="Persons 65 years old and over, Male, MOE,&_years_dash "
+    mPop65andOverYearsF_&_years.="Persons 65 years old and over, Female, MOE,&_years_dash "
 
     mhshldincunder10000_&_years. = "Household income under 10000 in the past 12 months, total, MOE, &_years_dash"
 	mhshldinc10000to14999_&_years.= "Household income from 10000 to 14999 in the past 12 months, total, MOE, &_years_dash"
```
Comment (lhendey) [inline Macros/ACS_summary_geo_source_tr_vars.sas]: please put these vars in the program with the rest of the ones one that topic.

File: Macros/ACS_summary_geo.sas (hunk 1 above)
Comment (rpitingolo) [reply Macros/ACS_summary_geo.sas]: @lhendey yes. I've deleted the duplicates.

File: Macros/ACS_summary_geo.sas (hunk 2 above)
Comment (rpitingolo) [reply Macros/ACS_summary_geo.sas]: @lhendey I'm not sure I understand what you mean by 'in the new code' ?

Comment (lhendey) [reply Macros/ACS_summary_geo.sas]: @rpitingolo  i i think i was just double check in your current version of summary_geo that those vars are there- it was unclear to me in the side by side comparison if they just got moved around or not.

Comment (rpitingolo) [reply Macros/ACS_summary_geo.sas]: @lhendey Oh OK. I see what you are saying. I'll check.

Comment (rpitingolo) [reply Macros/ACS_summary_geo.sas]: @lhendey they are still there. In the %let moe2013plus = list. That's where you want them, right?

Comment (lhendey) [reply Macros/ACS_summary_geo.sas]: @rpitingolo fine.

File: Macros/ACS_summary_geo_source_tr_vars.sas (hunk 4 above)
Comment (rpitingolo) [reply Macros/ACS_summary_geo_source_tr_vars.sas]: @lhendey OK, I've cleaned up ACS_summary_geo and ACS_summary_geo_source_tr_vars to put vars under their correct topics.

**Comment by rpitingolo** [comment]
@lhendey can you please take another look and run one of the summary programs to test?

**Comment by maggiesu0725** [comment]
@rpitingolo  Some StanC variables are not in the ACS summary file when I ran the program yesterday, I think I updated all the ACS programs to include those. Please let me know when all the conflicts are resolved and then we can talk about potentially adding some variables.

**Comment by lhendey** [comment]
@maggiesu0725 - can you give an example of something that's missing? cc @rpitingolo

**Comment by maggiesu0725** [comment]
pop35_64yearsM pop35_64yearsF pop18_34yearsF pop18_34yearsM .....

hshldincunder15000 hshldinc15000to34999 hshldinc35000to49999 hshldinc50000to74999 hshldinc200000plus 

@rpitingolo @lhendey

**Comment by maggiesu0725** [comment]
The code I added: /*Yipeng Su Added new varables 6/4/2018*/
  PopUnder18YearsM_&_years.=
  sum(B01001e3, B01001e4, B01001e5, B01001e6);   

  PopUnder18YearsF_&_years.= 
  sum(B01001e27, B01001e28, B01001e29, B01001e30);

  Pop18_34YearsM_&_years.=
  sum(B01001e7, B01001e8, B01001e9, B01001e10, B01001e11, B01001e12);

  Pop18_34YearsF_&_years.= 
  sum(B01001e31, B01001e32, B01001e33, B01001e34, B01001e35, B01001e36);

  Pop35_64YearsM_&_years.=
  sum(B01001e13, B01001e14, B01001e15, B01001e16, B01001e17, B01001e18, B01001e19);

  Pop35_64YearsF_&_years.= 
  sum(B01001e37, B01001e38, B01001e39, B01001e40, B01001e41, B01001e42, B01001e43);

  Pop65andOverYearsM_&_years.=
  sum(B01001e20, B01001e21, B01001e22, B01001e23, B01001e24, B01001e25);

  Pop65andOverYearsF_&_years.= 
  sum(B01001e44, B01001e45, B01001e46, B01001e47, B01001e48, B01001e49);
    hshldincunder10000_&_years. = B19001e2;
	hshldinc10000to14999_&_years.= B19001e3;
    hshldinc15000to19999_&_years.= B19001e4;
	hshldinc20000to24999_&_years.= B19001e5;
    hshldinc25000to29999_&_years.= B19001e6;
	hshldinc30000to34999_&_years.= B19001e7;
	hshldinc35000to39999_&_years.= B19001e8;
	hshldinc40000to44999_&_years.= B19001e9;
    hshldinc45000to49999_&_years.= B19001e10;
	hshldinc50000to59999_&_years.= B19001e11;
	hshldinc60000to74999_&_years.= B19001e12;
	hshldinc75000to99999_&_years.= B19001e13;
	hshldinc100000to124999_&_years.= B19001e14;
	hshldinc125000to149999_&_years.= B19001e15;
	hshldinc150000to199999_&_years.= B19001e16;
	hshldinc200000andover_&_years.= B19001e17;

are they still in here? @rpitingolo @lhendey

**Comment by rpitingolo** [comment]
@maggiesu0725 once we merge this branch and batch submit these will be in the 2012_16 file.  @lhendey

**Comment by lhendey** [comment]
@maggiesu0725 - those appear in the output i'm looking at.

@rpitingolo  cluster2017???

**Comment by rpitingolo** [comment]
@lhendey nice catch. I'll add cluster2017 now.

**Pull request merged**